project/hychiu02/README.md

```diff
@@ -0,0 +1,96 @@
+# AutoEncoder for arrhythmia detection
+
+## Basic information
+
+[GitHub](https://github.com/hychiu02/AutoEncoder-for-arrhythmia-detection)
+
+Arrhythmia dectection using ECG signal
+
+## Problem to solve
+Heart diseases are one of the significant reasons for death all over the planet. To perform arrhythmia detection, Electrocardiograms(ECGs) is used to monitor hte functioning of heart by capturing electrical activity. Because of needs of real-time inference, the model sizeand  parameters should be small, autoencoder seems to be a good choice. I want to develop a method to build, train and inference a MLP model with C++. 
+
+## Prospective Users
+People who want to build Multilayer Perceptron model for particular case.
+
+## System Architecture
+There will be a Node class for storing weight, a Layer class with Nodes for forward and backward weights calculation and a MLP class containing Layers  
+The flow will look like this

```

Comment:
I assume the data I/O is handled by using Python?

Reply:
Actually I plan to do only unit test on python. Is it ok for term project?

Reply:
It is OK.  But you did not answer my question.  Please think about how you will do I/O, which will be a hurdle.

Reply:
I just have raw idea to read this kind of text data by fopen or something. I'll try to survey how to hide I/O survey

Reply:
You may also list it as a limitation of the system and mark is as unhandled in the proposal.

Reply:
Ok, I added it to my proposal. Thank you!